cmd/vic-machine/list/list.go

```diff
@@ -114,11 +114,10 @@ func (l *List) prettyPrint(op trace.Operation, cli *cli.Context, vchs []*vm.Virt
 			version = vchConfig.Version.ShortVersion()
 			upgradeStatus = l.upgradeStatusMessage(op, vch, installerVer, vchConfig.Version)
 		}
-
-		parentPath := path.Dir(path.Dir(vch.InventoryPath))
-		name := path.Base(vch.InventoryPath)
```

Comment:
Line 103 of `vch_list_get.go` seems very similar to this line. Should it be updated as well?

https://github.com/vmware/vic/blob/ca108842bf23d56344c7d503b3be6f6889eb2b2b/lib/apiservers/service/restapi/handlers/vch_list_get.go#L103

Reply:
when we call name on the object in `list.go` it's just doing a `path.Base` behind the scenes...I changed in `list.go` because I thought it was easier to read / follow.  Happy to change here if you'd like...